third_party/tensorrt/tensorrt_configure.bzl

```diff
@@ -22,102 +22,96 @@ _TF_NEED_TENSORRT = "TF_NEED_TENSORRT"
 
 _TF_TENSORRT_LIBS = ["nvinfer", "nvinfer_plugin"]
 _TF_TENSORRT_HEADERS = ["NvInfer.h", "NvUtils.h", "NvInferPlugin.h"]
+_TF_TENSORRT_HEADERS_V2 = ["NvInfer.h", "NvUtils.h", "NvInferPlugin.h",
+                           "NvInferVersion.h", "NvInferRTSafe.h",
+                           "NvInferRTExt.h", "NvInferPluginUtils.h"]
 
 _DEFINE_TENSORRT_SONAME_MAJOR = "#define NV_TENSORRT_SONAME_MAJOR"
 _DEFINE_TENSORRT_SONAME_MINOR = "#define NV_TENSORRT_SONAME_MINOR"
 _DEFINE_TENSORRT_SONAME_PATCH = "#define NV_TENSORRT_SONAME_PATCH"
 
 def _at_least_version(actual_version, required_version):
-    actual = [int(v) for v in actual_version.split(".")]
-    required = [int(v) for v in required_version.split(".")]
-    return actual >= required
-
-def _update_tensorrt_headers(tensorrt_version):
-    if not _at_least_version(tensorrt_version, "6"):
-        return
-    _TF_TENSORRT_HEADERS = [
-        "NvInferVersion.h",
-        "NvInfer.h",
-        "NvUtils.h",
-        "NvInferPlugin.h",
-        "NvInferRTSafe.h",
-        "NvInferRTExt.h",
-        "NvInferPluginUtils.h",
-    ]
+  actual = [int(v) for v in actual_version.split(".")]
```

Comment:
Please revert all indention changes in this file. .bzl files requires 4-space indention.